GRPCServiceA/pkg/utils/utils.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"strings"
+	"time"
+
+	"github.com/google/uuid"
+)
+
```

Comment:
you can separate this utils, in  provider utils (provide you and specific stufs, Uid, Time, etc) and validations (Email, PwdHash, etc) this to do a readable code.